src/main/java/org/jabref/logic/cleanup/CleanupWorker.java

```diff
@@ -86,4 +103,17 @@ private CleanupJob toJob(CleanupPreferences.CleanupStep action) {
                     throw new UnsupportedOperationException(action.name());
         };
     }
+
+    private void showFileMoveExceptions(BibEntry entry) {
+        StringBuilder sb = new StringBuilder();
+        String title = entry.getTitle().orElse(Localization.lang("Unknown Title"));
```

Comment:
Normally we use the citaton key. You can use `org.jabref.model.entry.BibEntry#getAuthorTitleYear(int)` with `40` as `maxCharacters` as fallbeack.

Reply:
I deleted the citation key part as I think it'd be difficult passing both the entry and the exceptions from `org.jabref.logic.cleanup.CleanupWorker` to `org.jabref.gui.cleanup.CleanupAction` (I will have to create a wrapper to put the entry and its corresponding exceptions)